schema/verify/functions/age_conversion.sql

```diff
@@ -0,0 +1,30 @@
+-- Verify seattleflu/schema:functions/age_conversion on pg
+
+begin;
+
+do $$ 
+
+	declare
+    	age_in_months int;
+    	months int;
+    	years numeric;
+    	month_to_years numeric;
+
+	begin
+
+		with ages (age_as_interval, age_in_months) as (
+		    select make_interval(months => x), x from generate_series(0, 12) as x
+		    union
+		    select make_interval(years => x), x * 12 from generate_series(1, 6) as x
+		)		
+		select ages.age_in_months, age_in_months(age_as_interval), age_in_years(age_as_interval), age_month_to_years(ages.age_in_months) 
+		  into age_in_months, months, years, month_to_years
+		  from ages;
```

Comment:
This only tests a single row of the `ages` table. In PL/pgSQL (the language inside this `do` block), it's good to use the  `select … into strict …` form of `select … into` so that an error is thrown if more than one (or no) rows are returned.

You can loop over the results of a query and assert on each row's values using a for loop: https://www.postgresql.org/docs/10/plpgsql-control-structures.html#PLPGSQL-RECORDS-ITERATING